engine/worker/internal/action/builtin_artifact_upload.go

```diff
@@ -80,18 +82,38 @@ func RunArtifactUpload(ctx context.Context, wk workerruntime.Runtime, a sdk.Acti
 		go func(path string) {
 			log.Debug(ctx, "worker.RunArtifactUpload> Uploading %s projectKey:%v integrationName:%v job:%d", path, projectKey, integrationName, jobID)
```

Comment:
in fact, now you don't need to write `worker.RunArtifactUpload>` because this is automatically added as field in the logs